packages/core/pluggableElementTypes/models/BaseTrackModel.ts

```diff
@@ -130,21 +129,6 @@ export function createBaseTrackModel(
       setMinimized(flag: boolean) {
         self.minimized = flag
       },
-      /**
-       * #action
-       */
-      activateConfigurationUI() {
```

Comment:
appeared unused, I think the behavior is wrong also (getting return value of editTrackConfiguration and then hiding/showing...)